specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -15,36 +15,25 @@
 - [Alternatives](#alternatives)
 
 ## Summary
-
-<!--
-Short summary of the proposal. It will be used as user-focused
-documentation such as release notes or a (customer facing) development roadmap.
-The tone and content of the `Summary` section should be
-useful for a wide audience.
--->
+At the moment customers can deploy and manage jobs through the VDK CLI tool or through the API's.
+We want to extend on this and create a web frontend for the VDK.
+The frontend will let customers, create,delete deploy and generally managed nad browse their jobs.
+Frontend is written in an extensible format to easily allow other teams to build extensions.
+To achieve this extensibility the frontend will be built in angular.
 
 ## Glossary
-<!--
-Optional section which defines terms and abbreviations used in the rest of the document.
--->
+data-pipelines: This is the npm project of the bootstrapping codebase it contains all the vdk specific frontend code. for more details please see [README.md](/projects/frontend/data-pipelines/README.md)
+shared-components, @vdk/shared: This is a set of components that are used within the data-pipelines project but are generic and could be re-used by other projects [README.md](/projects/frontend/data-pipelines/README.md)
 
 ## Motivation
-<!--
-It tells **why** do we need X?
-Describe why the change is important and the benefits to users.
-Explain the user problem that need to be solved.
--->
+We want to improve the usability of VDK. With a UI customers will be able to use VDK much quicker and much more proficiently because UIs are easier to understand and become familiar with than CLI tools or APIs.
 
-## Requirements and goals
-<!--
-It tells **what** is it trying to achieve?
-List the specific goals (functional and nonfunctional requirements)? How will we
-know that this has succeeded?
 
-Specify non-goals. Clearly, the list of non-goals can't be exhaustive.
-Non-goals are only features, which a contributor can reasonably assume were a goal.
-One example is features that were cut during scoping.
--->
+## Requirements and goals
+1. It should be easy to setup and run locally or in a k8s cluster
+2. It should be easy to extend with new features
```

Comment:
what is this extensibility? Sounds very generic. How do you measure if extensible or not, what's the criteria.

Reply:
I don't think extensibility is generic for frontends.
Most of the time it wouldn't be trivial to add a new screen to an existing GUI. 

How I would measure.
1. Is adding a new screen documented?
2. Do we expect programmers who didn't work on the initial build of the project to be able to build a section for their needs in  isolated manner
3. After they have created a section can they easily add it to a deployed instance?

Reply:
My suggestion here is to postpone the extensibility statements from the VEP - since it's still an open question, and it would require a dedicated focus to resolve it in my opinion.
I think the Use Cases should rather be a priority at this early point.

Reply:
@ivakoleva  I added details below. 
Can you read and let me know what you think!